src/library/basetracktablemodel.cpp

```diff
@@ -782,6 +782,7 @@ QVariant BaseTrackTableModel::roleValue(
         case ColumnCache::COLUMN_LIBRARYTABLE_BPM: {
             bool ok;
             const auto bpmValue = rawValue.toDouble(&ok);
+            // FIXME: calling bpm.value() without checking bpm.isValid()
             return ok ? bpmValue : mixxx::Bpm().value();
```

Comment:
mixxx::Bpm().value() should be replaced with mixxx::Bpm::kValueUndefined wherever it occurs